### ISSUE TITLE
Videos not working

### ISSUE THREAD

**Issue by mkhalila** [yaquawa/jquery.photoswipe#14]
I've tried a few ways to add video but none of them are working. Any help? Images currently working great

**Comment by rustykowski** [comment]
Yes, I also just failed at videos, too bad. Everything else is fine, including the zoom effect.
